ArduPlane/quadplane.cpp

```diff
@@ -3675,6 +3690,9 @@ bool QuadPlane::using_wp_nav(void) const
 MAV_TYPE QuadPlane::get_mav_type(void) const
 {
     if (mav_type.get() == 0) {
+        if (enabled()) {
```

Comment:
This would have to be:
```suggestion
        if (available()) {
```

Reply:
Thank you very much!!